src/Files.App/Filesystem/FilesystemOperations/ShellFilesystemOperations.cs

```diff
@@ -27,6 +27,8 @@ public class ShellFilesystemOperations : IFilesystemOperations
 
 		private FilesystemOperations filesystemOperations;
 
+		private RecycleBinHelpers recycleBinHelpers;
+
```

Comment:
`RecycleBinHelpers` is now static

Reply:
Oh true (doesn't notice even though he did it)